packages/application-generic/src/services/queue.service.ts

```diff
@@ -22,10 +21,9 @@ export class QueueService<T = unknown> {
   public readonly bullMqService: BullmqService;
   public readonly DEFAULT_ATTEMPTS = 3;
 
-  constructor() {
+  constructor(public name = 'standard') {
```

Comment:
We shouldn't allow to override this property. Why is it needed? this can lead to some really bad bugs and back compatibility issues.

Reply:
explained in the first comment in `MetricQueueService` file